packages/core-js/.mocharc.json

```diff
@@ -1,5 +1,4 @@
 {
   "spec": ["test/**/*.test.js"],
-  "parallel": true,
```

Comment:
Being unit tests in JS these tests are actually faster running in sync (and allows us to use `.only` for debugging)